ionoscloud/data_source_autoscaling_group.go

```diff
@@ -0,0 +1,300 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	autoscaling "github.com/ionos-cloud/sdk-go-vm-autoscaling"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services"
+	autoscalingService "github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/autoscaling"
+)
+
+func dataSourceAutoscalingGroup() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceAutoscalingGroupRead,
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:         schema.TypeString,
+				Description:  "UUID of the Autoscaling Group.",
+				Optional:     true,
+				ValidateFunc: validation.All(validation.IsUUID),
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "User-defined name for the Autoscaling Group.",
+				Optional:    true,
+			},
+			"max_replica_count": {
+				Type:        schema.TypeInt,
+				Description: "Maximum replica count value for `targetReplicaCount`. Will be enforced for both automatic and manual changes.",
+				Computed:    true,
+			},
+			"min_replica_count": {
+				Type:        schema.TypeInt,
+				Description: "Minimum replica count value for `targetReplicaCount`. Will be enforced for both automatic and manual changes.",
+				Computed:    true,
+			},
+			"target_replica_count": {
+				Type:        schema.TypeInt,
+				Description: "The target number of VMs in this Group. Depending on the scaling policy, this number will be adjusted automatically. VMs will be created or destroyed automatically in order to adjust the actual number of VMs to this number. If targetReplicaCount is given in the request body then it must be >= minReplicaCount and <= maxReplicaCount.",
+				Computed:    true,
+			},
+
+			"policy": {
+				Type:        schema.TypeList,
+				Description: "Specifies the behavior of this Autoscaling Group. A policy consists of Triggers and Actions, whereby an Action is some kind of automated behavior, and a Trigger is defined by the circumstances under which the Action is triggered. Currently, two separate Actions, namely Scaling In and Out are supported, triggered through Thresholds defined on a given Metric.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"metric": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "The Metric that should trigger the scaling actions. Metric values are checked at fixed intervals.",
+						},
+						"range": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "Defines the time range, for which the samples will be aggregated. Default is 120s.",
+						},
+						"scale_in_action": {
```

Comment:
the `"delete_volumes"` attribute is missing `scale_in_action` of the data source